gen/go/metrics/v1/metrics.pb.go

```diff
@@ -1077,7 +1077,7 @@ func (m *SummaryValue) GetPercentileValues() []*SummaryValue_ValueAtPercentile {
 // Represents the value at a given percentile of a distribution.
 type SummaryValue_ValueAtPercentile struct {
 	// The percentile of a distribution. Must be in the interval
-	// (0.0, 100.0].
+	// [0.0, 100.0].
```

Comment:
This is a real (though harmless) example that currently in master generated Go files are out of sync.